airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```
@@ -2398,3 +2398,10 @@ interpolation:
       return_type: str
       examples:
         - "{{ 'ZmFrZSByZWZyZXNoX3Rva2VuIHZhbHVl' | base64decode }} -> 'fake refresh_token value'"
+    - title: String
+      description: Converts the specified value to a string
```

Nit: other descriptions seem to have the period in the end. 

```suggestion
      description: Converts the specified value to a string.
```